rawsnippets/pipeline-with-buildah-task.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
It would be better if you combine pipeline-with-buildah-task.yaml, pipeline-with-kubectl-task.yaml  and pipeline-with-maven-task.yaml in to one template